src/main/java/com/github/bartimaeusnek/bartworks/common/tileentities/multis/GT_TileEntity_ExtremeIndustrialGreenhouse.java

```diff
@@ -84,6 +84,7 @@
     private boolean isIC2Mode = false;
     private byte glasTier = 0;
     private int waterusage = 0;
+    private static boolean isNoHumidity = false;
```

Comment:
Remove static modifier, you shouldn't be able to change the mode for diffrent people EIGs

```suggestion
    private boolean isNoHumidity = false;
```